packages/web-components/fast-foundation/src/accordion-item/accordion-item.template.ts

```diff
@@ -28,8 +28,8 @@ export const accordionItemTemplate: FoundationElementTemplate<
                 id="${x => x.id}"
                 @click="${(x, c) => x.clickHandler(c.event as MouseEvent)}"
             >
-                <span class="heading">
-                    <slot name="heading" part="heading"></slot>
+                <span class="heading-content" part="heading-content">
```

Comment:
I'm open to an updated name here - my thought is that "content" could be confusing in itself because the accordion item has "content".